src/show/components/Navbar/view.jsx

```diff
@@ -81,7 +99,7 @@ const Navbar = ({ currentUser, signIn, signOut, signUp }) => {
           {currentUser?.isLoggedIn ? (
             <div className='flex transition-all duration-300 items-center hover:px-2 hover:py-1'>
               <Button
-                onClick={() => setOpenLogOutModal(true)}
+                onClick={() => toggleLogOutModal()}
```

Comment:
```suggestion
                onClick={toggleLogOutModal}
```

same bellow, if there's no argument to be passed in this onClick you can do it like that ^

Reply:
done